e2e/data-factory/README.md

```
@@ -47,73 +43,73 @@ This is part of the Ghost e2e test suite. All dependencies are managed by the ma
 
 ### In Tests
 
-```javascript
-// Import from the built files
-const {PostFactory, KnexPersistenceAdapter} = require('../build/data-factory');
+**Option 1: Use setup helpers (recommended)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)